src/controller/RacingGameController.js

```diff
@@ -0,0 +1,72 @@
+const RacingGame = require('../model/RacingGame');
+const inputView = require('../view/inputView');
+const outputView = require('../view/outputView');
+const validate = require('../utils/validation');
+const { SIGN, ERROR } = require('../utils/constants');
+
+class RacingGameController {
+  #inputView;
+  #outputView;
+  constructor() {
+    this.#inputView = inputView;
+    this.#outputView = outputView;
+    this.initiate();
+  }
+
+  initiate() {
+    this.#readCarNames();
+  }
+
+  #readCarNames() {
+    this.#inputView.readUserInputNames((input) => {
+      const names = input.split(SIGN.comma);
```

Comment:
```suggestion
      const names = input.split(',');
```
`SIGN.comma`가 애매하네 `,`는 사실상 바뀔리가 없으니까 그냥 콤마로 해도 될 듯

Reply:
그런 점에서 split 하는 위치도 살짝 애매하긴 함.
split을 한다는 건 input을 formatting 한다는 거잖아. 그럼 충분히 분리해서 관리할 필요가 있을것 같다. formatting 하는 방식은 추후에 변경될 수도 있는거니까 .(ex: _로 하는 구분도 추가 한다던지, 공백으로 구분하는 것도 추가 한다던지 등등) 

split하는 위치를 this.#handleCarNamesException(names) 내부로 옮기는 것도 나쁘지 않다고 생각해.
이름은 `handleCarNamesInput(input)` 정도로 바꾸는거지